manager/container.go

```diff
@@ -78,6 +83,17 @@ type containerData struct {
 	collectorManager collector.CollectorManager
 }
 
+// jitter returns a time.Duration between duration and duration + maxFactor * duration,
+// to allow clients to avoid converging on periodic behavior.  If maxFactor is 0.0, a
+// suggested default value will be chosen.
+func jitter(duration time.Duration, maxFactor float64) time.Duration {
```

Comment:
I think we should keep the function parameter at minimum, and if we want to remove the constant and simply default the argument that's fine.  But at this point we're splitting hairs.